src/app/core/coming-soon/coming-soon/coming-soon.component.ts

```diff
@@ -9,17 +9,31 @@ import {
 import { AlertService } from "../../alerts/alert.service";
 import { ActivatedRoute } from "@angular/router";
 import { AnalyticsService } from "../../analytics/analytics.service";
-import { MAT_DIALOG_DATA } from "@angular/material/dialog";
+import { MAT_DIALOG_DATA, MatDialogModule } from "@angular/material/dialog";
+import { DialogCloseComponent } from "../../common-components/dialog-close/dialog-close.component";
+import { MatButtonModule } from "@angular/material/button";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { NgIf } from "@angular/common";
+import { RouteTarget } from "../../../app.routing";
 
 /**
  * Placeholder page to announce that a feature is not available yet.
  *
  * This integrates with analytics and allows user to explicitly request the feature.
  */
+@RouteTarget("ComingSoon")
```

Comment:
Should we replace the `RouteTarget` decorator with the `DynamicComponent` decorator as it is has no special meaning any more. In fact, both the registries are not really needed anymore as the lookup can happen through the global `dynamicComponents` map, which holds the lazy import functions. The `DynamicComponent` decorator has the benefit, that after the first loading, the component can be used without another lazy loading. What performance impact this has - I am not sure.